homework/hackyourtemperature/server.js

```diff
@@ -0,0 +1,16 @@
+import express, { urlencoded } from "express";
+
+const app = express();
+
+app.use(express.json());
+app.use(urlencoded({ extended: false }));
+app.get("/", (req, res) => {
+  res.send("<h1>Hello from backend to frontend!</h1>");
+});
+app.post("/weather", (req, res) => {
+  const cityName = req.body;
```

Comment:
If you want to do this you should use destructuring like this
```suggestion
  const { cityName } = req.body;
```

Then the user can send a request like:

```json
{
  "cityName": "Amsterdam"
}
```